scripts/copyright.js

```diff
@@ -1,15 +0,0 @@
-/*
- * Copyright <%= YEAR %> Spotify AB
```

Comment:
Noticed you're not using `<%= YEAR %>` in `@spotify/eslint-config-oss`, is that a deliberate choice?

Reply:
Argh it most certainly is not! I know exactly what happened there and will release a patch fix.